test/fixtures/.shout/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('extend')(require('../../../defaults/config.js'), {
+	prefetch: true
+});
```

Comment:
Indeed, but I don't expect this config loading to stay there before other tests are added, and we won't have to do that config rain dance to satisfy the test.